_posts/2021-02-03-episode-three-of-lets-talk-about-public-code.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Episode three of Let's talk about public code! - Sergei Maertens and Tjerk Vaags from OpenZaak"
+date: 2021-02-03
+author: Jan Ainali
+type: blogpost
+excerpt: The third episode of our podcast is available
+categories:
+  - news
+---
+
+# Episode three of Let's talk about public code!
+
+In the third episode we talked to Sergei Maertens from Maykin Media and Tjerk Vaags from Contezza.
+How can you design a process for handling Common Vulnerabilities and Exposures worked in a collaborative codebase?
```

Comment:
Question: 'worked' doesn't work. Do you mean it to indicate a tense change (for example "a process that works in a...", or is there a word missing (for example, "worked up in a collaborative codebase")?

Reply:
I was thinking the first, something that was functional.

Reply:
Oh, now I see my error, I rewrote this as a question but was doing a sloppy job.